kwikapi/api.py

```diff
@@ -520,6 +520,8 @@ def handle_request(self, request):
                 self._invoke_pre_call_hook(request)
                 result = request.fn(**request.fn_params)
 
+                protocol = self._find_request_protocol(request)
```

Comment:
@RamanjaneyuluIdavalapati What is this protocol used for??

Reply:
@jaswanth098 We use that protocol to serialise the response/result.
By default JSON protocol will be used to serialise and deserialise the incoming(client sends data to the server) and outgoing(server sends data to the client) data.

We can change this default protocol. [Ref](https://github.com/deep-compute/kwikapi/blob/master/README.md#kwikapi-also-supports-defining-specific-protocol-for-specific-request)

But this protocol will be used to serialise and deserialise for both incoming and outgoing data.
I have supported here such that the user can change the protocol for outgoing data.

Reply:
got it thanks for clarification